inst/builtin/bs5/shiny/_mixins.scss

```diff
@@ -4,9 +4,5 @@ $accordion-button-active-bg: null;
 $accordion-button-active-color: null;
 $accordion-icon-active-color: null;
 
-// Shiny: selectInput() and selectizeInput()
-$selectize-color-dropdown-item-active: $component-active-bg !default;
-$selectize-color-dropdown-item-active-text: $component-active-color !default;
-
```

Comment:
Why is this getting removed?

Reply:
It doesn't do anything -- the correct sass variable is `$select-color-dropdown` -- which indicates that it isn't needed.